app/models/response_weight.rb

```diff
@@ -0,0 +1,15 @@
+class ResponseWeight < ActiveRecord::Base
+  attr_accessible :response1_id, :response2_id, :weight
+  def self.find_weight(response1, response2)
+    if response1 == response2
+        return 0
+    end
+    query1 = ResponseWeight.where(:response1_id => response1.to_i, :response2_id => response2.to_i)
```

Comment:
This is a weird way to do it...this would preferably be done with a scope (I think?) or at least a single query abstracted out as a static method in ResponseWeight. If there's no time to refactor this, let's keep this in mind for later this week.